wolfssl/wolfcrypt/types.h

```diff
@@ -597,7 +597,11 @@ decouple library dependencies with standard string, memory and so on.
 
                         if ((int)bufsize <= 0) return -1;
                         va_start(ap, format);
+                    #ifndef _WIN32_WCE
```

Comment:
Can this be fixed to use our `XVSNPRINTF` macro and changed in wc_port.h?

Reply:
Absolutely!